### ISSUE TITLE
NVDA is not announcing tooltip information present for “Type File name” edit field: A11y_VisualStudioCodeClient_SideBarOptions_Accessing The Explorer side bar options and Creating a new file_ScreenReader.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#250986]
Please do not close this bug. This bug should only be closed by Trusted Tester after verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_VisualStudioCodeClient_Win32_Jun2025; #Visual Studio Code Client; #WCAG1.3.1; #FTP; #Info and Relationship; #AILimited; #Win11; #Benchmark;

### Environment Details:
App Name: Visual Studio Code Client 
#URL: Visual Studio Code Insider  
Microsoft Windows 11 Enterprise 
OS Build: 26100.4202
NVDA version: 2024.4.2(2024.4.2.35031)

### Repro Steps:

1. Open Visual Studio Code Insider editor. 
2. Press CTRL + SHIFT + E to open side navigation. 
3. Start NVDA and TAB till “New file” control and hit ENTER key. 
4. Observe the issue whether NVDA is announcing tooltip information present for “Type File name” edit field or not. 

### Actual Result:
NVDA is not announcing tooltip information present for “Type File name” edit field.
**Note: **This issue is not observed with JAWS.****

### Expected Results:
NVDA should announce tooltip information present for “Type File name” edit field.

### User Impact:
Screen reader users will not get tooltip information present for “Type File name” edit field.

### Attachment:

![Image](https://github.com/user-attachments/assets/5618f780-820c-43ea-9746-8210377fe97d)

The below screen shot illustrate that Issue is No repro with JAWS.
![Image](https://github.com/user-attachments/assets/f9557e7a-5ec0-4aae-a546-e5506f854c6d)

**Comment by meganrogge** [comment]
If the issue is not happening with JAWS, that suggests this is either a feature of JAWS or is an NVDA issue -- not a VS Code issue, thus closing.

**Comment by keerthiduvvuri** [comment]
Discussed this bug in bug discussion call, Screen reader is announcing as expected only, Hence closing the bug.
